include/h2o.h

```diff
@@ -989,9 +989,16 @@ typedef struct st_h2o_conn_callbacks_t {
      */
     uint32_t (*num_reqs_inflight)(h2o_conn_t *conn);
     /**
-     * optional callbacks that returns the tracer regsitry
+     * optional callbacks that return the tracer registry
      */
     quicly_tracer_t *(*get_tracer)(h2o_conn_t *conn);
+    /**
+     * An optional callback reporting an estimate of the connection round-trip
+     * time in microseconds obtained by timing the acknowledgement of our
+     * settings frame in HTTP/2.  Only supported for H2.  The callback will
+     * return a negative value if the information is not yet available.
```

Comment:
I think we might want to be talk about the concept in a version-agnostic way (that the RTT being reported is at HTTP-level rather than at transport-level), and explain how it is implemented specifically for HTTP/2. Hence something like:
```suggestion
     * An optional callback reporting an RTT estimate between the HTTP server and the HTTP client, measured in microseconds. At the
     * moment, this callback is available only for HTTP/2. For HTTP/2, time difference between when the SETTINGS frame was sent and
     * when a SETTINGS-ack was received is used as the estimate. The callback will return a negative value if the information is not
     * yet available.
```